.environment-scripts/install_local_packages.sh

```diff
@@ -4,6 +4,11 @@ CONDA_ENV=$1
 
 source activate $CONDA_ENV
 
+# we want to force a rebuild in case numpy version changes
+# this doesn't rebuild automatically when dependencies change version
+rm -f external/vcm/vcm/mappm.*.so
```

Comment:
I could see the "*" causing some funny business:
```suggestion
rm -f "external/vcm/vcm/mappm.*.so"
```